en/03_Drawing_a_triangle/01_Presentation/01_Swap_chain.adoc

```diff
@@ -244,46 +186,57 @@ VkSurfaceFormatKHR chooseSwapSurfaceFormat(const std::vector<VkSurfaceFormatKHR>
 
 === Presentation mode
 
-The presentation mode is arguably the most important setting for the swap chain, because it represents the actual conditions for showing images to the screen.
+The presentation mode is arguably the most important setting for the swap chain,
+because it represents the actual conditions for showing images to the screen.
 There are four possible modes available in Vulkan:
 
-* `VK_PRESENT_MODE_IMMEDIATE_KHR`: Images submitted by your application are transferred to the screen right away, which may result in tearing.
-* `VK_PRESENT_MODE_FIFO_KHR`: The swap chain is a queue where the display takes an image from the front of the queue when the display is refreshed and the program inserts rendered images at the back of the queue.
-If the queue is full then the program has to wait.
-This is most similar to vertical sync as found in modern games.
-The moment that the display is refreshed is known as "vertical blank".
-* `VK_PRESENT_MODE_FIFO_RELAXED_KHR`: This mode only differs from the previous one if the application is late and the queue was empty at the last vertical blank.
-Instead of waiting for the next vertical blank, the image is transferred right away when it finally arrives.
-This may result in visible tearing.
+* `VK_PRESENT_MODE_IMMEDIATE_KHR`: Images submitted by your application are
```

Comment:
Should  (or could?) use the vulkan.hpp enums instead. I think we should use them everywhere instead of the C constant names, as that is confusing to the reader.